ote/src/cljs/ote/ui/form_fields.cljs

```diff
@@ -142,18 +145,42 @@
       {:on-key-press #(when (= "Enter" (.-key %))
                         (on-enter))}))])
 
+(defmethod field :file [{:keys [update! label name max-length min-length regex
+                                 focus on-focus form? error warning table? full-width?
+                                 style input-style hint-style on-change]
+                          :as   field} data]
+[:div.upload-btn-wrapper
+ [:button {:style
+           {:padding "10px 20px 10px 20px"
+            :text-transform "uppercase"
+            :color "#FFFFFF"
+            :background-color "#1565C0"
+            :font-size "12px"
+            :font-weight "bold"}}
+           "Lisää csv" ]
+  [:input {:type "file"
+           :name name
+           :on-change
+            (if on-change
+              on-change
+              #(let [v %2]
```

Comment:
I don't think we need regex support to file input? Wouldn't you always want to have an `:on-change` option?

Reply:
on-change is what we want. Thanks.